bibtex-actions.el

```diff
@@ -122,6 +137,40 @@ manager like Zotero or JabRef."
     map)
   "Keymap for 'bibtex-actions'.")
 
+;;; Org-cite citation function
+
+(defun bibtex-actions--format-citation-org (keys)
+  "Format org-cite citations for the entries in KEYS.
+
+The full citation syntax is:
+
+  [cite/style:common prefix ;prefix @key suffix; ... ; common suffix]
+
+Everything is optional, except the brackets, 'cite' and the colon.
+Also the citation must contain at least a key.
+So its minimal form is:
+
+  [cite:@key]
+
+Note that the prefix here is for the citation as a whole; if you
+need to add one specific to an individual citation item, you
+would need to add that after inserting.
+
+  [cite:see ;item prefix @key]
+
+The same is true for suffixes like page numbers, which are
+specific to the item, rather than the citation as a whole.
+
+  [cite: see ;@key pp23-24]"
+  (let* ((prefix  (if bibtex-completion-cite-prompt-for-optional-arguments (read-from-minibuffer "Prefix: ") ""))
+         (styles bibtex-actions-org-cite-styles)
+         (style  (if bibtex-completion-cite-prompt-for-optional-arguments
+                     (ido-completing-read "Style: " styles nil nil nil nil "default")))
+         (prefix  (if (string= "" prefix)  "" (concat prefix  " ;")))
+         (style  (if (string-equal style "default") "" (concat "/" style))))
+    ;; this is derived from the pandoc syntax function, but this has two-levels of affixes
+    (format "[cite%s:%s%s]" style prefix (s-join ";" (--map (concat "@" it) keys)))))
```

Comment:
suggested change:
```elisp
(format "[cite%s: %s%s]" style prefix (s-join ";" (--map (concat "@" it) keys)))))
```

WDYT?

Reply:
I updated your comment to use the github "suggested" feature; note the difference in the source. You can access it by clicking the left most icon on the toolbar.

But on the substance, I don't believe so. Wouldn't that result in `[cite: @doe]`?